Sources/Core/MediaType.swift

```diff
@@ -198,6 +198,8 @@ extension MediaType {
     public static let multipart = MediaType(type: "multipart", subType: "mixed")
     /// JSON media type.
     public static let json = MediaType(type: "application", subType: "json", parameters: ["charset": "utf-8"])
+    /// JSON API media type (see https://jsonapi.org/format/).
+    public static let jsonApi = MediaType(type: "application", subType: "vnd.api+json", parameters: ["charset": "utf-8"])
```

Comment:
Acronyms that are commonly capitalized in English should be capitalized in Swift: `jsonAPI`.

Reply:
That's actually how I prefer it, I somehow thought it was not common in Swift :-) Done.